### ISSUE TITLE
Blazor: Minify CSS Isolation output

### ISSUE THREAD

**Issue by francescocristallo** [dotnet/aspnetcore#59644]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

When using CSS Isolation in Blazor, a CSS link elements is used:

`<link href="@Assets["{ASSEMBLY NAME}.styles.css"]" rel="stylesheet">`

Resulting CSS is NOT minified. Is not either a file that can be manually minified.

### Describe the solution you'd like

CSS resulting from CSS Isolation feature should be minimized by default

**Comment by oussamatecnique** [comment]
is there an official documentation that says that isolated css files are minimized by default? or that you would like it as default feature?
about a workarround you can add a posteventbuild to minify

**Comment by francescocristallo** [comment]
> is there an official documentation that says that isolated css files are minimized by default? or that you would like it as default feature? about a workarround you can add a posteventbuild to minify

Yes, I opened the feature request to suggest adding it as a default feature

**Comment by javiercn** [comment]
@francescocristallo thanks for contacting us.

This is not something that we plan to implement any time soon. Based on our experiments at the time, we found that applying minification didn't yield any significant further savings after serving the compressed file with gzip/brotli, hence we don't think it's worth the investment.
